src/spikeinterface/sortingcomponents/benchmark/benchmark_motion_interpolation.py

```diff
@@ -101,20 +112,32 @@ def __init__(
     def recordings(self):
         if self._recordings is None:
             self._recordings = {}
+
             for key in (
                 "drifting",
                 "static",
             ):
                 rec, _ = read_mearec(self.mearec_filenames[key])
+                self._recordings["raw_" + key] = rec
+
                 if self.do_preprocessing:
-                    rec = bandpass_filter(rec)
-                    rec = common_reference(rec)
+                    # all computation are done in float32
+                    # rec = bandpass_filter(rec)
+                    rec = common_reference(rec, dtype="float32")
+                    rec = highpass_filter(rec, freq_min=150.0)
```

Comment:
Highpass should come first!

Reply:
yes. you are right
for mearec this is not so important.

Reply:
In fact this was to mimic exactly what KS2.5 is doing.
I will add a comment in the code.

Reply:
I puit more doc in the code